### PR TITLE
docs(aws): Add documentation re AWS deployments

### PR DESCRIPTION
**Pull request by rorymchugh**
- Add `docs/deployment-aws.md`
- Add reference to new file in `docs/index.md`

### CODE REVIEW

File: docs/deployments-aws.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+- ECS Workloads: https://github.com/cambridge-collection/terraform-aws-workload-ecs
+
+## Container Images
+
+The ECS workloads and several lambdas are dependent on container images deployed to the AWS Elastic Container Repository (ECR). Images are managed outside of Terraform. The following container images are used:
+
+- Lambdas:
+    - SOLR listener lambdas: https://github.com/cambridge-collection/cudl-solr-listener
+    - TEI Processing: https://github.com/cambridge-collection/cudl-data-processing-xslt
+    - Transkribus:
```
Comment (jlf44) [inline docs/deployments-aws.md:28]: - Other functions, incl. Transkribus: https://github.com/cambridge-collection/data-lambda-transform

Comment (jlf44) [inline docs/deployments-aws.md]: I believe it's:
- Transkribus: https://github.com/cambridge-collection/curious-cures-transkribus-lambda

**Review by jlf44 — APPROVED** [review]
Looks really good

File: docs/deployments-aws.md (hunk 1 above)
Comment (rorymchugh) [reply docs/deployments-aws.md]: aha that's the one thanks

**Pull request merged**